api/api/clubs/urls.py

```diff
@@ -4,5 +4,7 @@
 
 urlpatterns = [
     path('', views.ListClubsView.as_view()),
-    path('<int:pk>/', views.UpdateClubsView.as_view())
+    path('<int:pk>/', views.UpdateClubsView.as_view()),
+    path('<int:pk>/register/', views.CreateClubRegistrationRequestView.as_view()),
```

Comment:
here url should be `<int:pk>/registration-requests/`

This document should be helpful in understanding what the urls should be: https://restfulapi.net/resource-naming/